microsoft-edge/webview2/get-started/hololens2.md

```diff
@@ -318,87 +260,87 @@ Now that you've reviewed the `WebView` code, let's extend the functionality of o
+   public class WebViewBrowser: MonoBehaviour
+   {
+      // Declare UI elements: Back button, Go button, and URL input field
+      public Button BackButton;
+      public Button GoButton;
+      public TMP_InputField URLField;
 
+      private void Start()
+      {
+         // Get the WebView component attached to the game object
+         var webViewComponent = gameObject.GetComponent<WebView>();
+         webViewComponent.GetWebViewWhenReady((IWebView webView) =>
+         {
+               // If the WebView supports browser history, enable the Back button
+               if (webView is IWithBrowserHistory history)
+               {
+                  // Add an event listener for the Back button to navigate back in history
+                  BackButton.onClick.AddListener(() => history.GoBack());
 
-<!-- ------------------------------ -->
-#### Remote Connection
+                  // Update the Back button's enabled state based on whether there's any history to go back to
+                  history.CanGoBackUpdated += CanGoBack;
+               }
 
-To set up your remote connection:
+               // Add an event listener for the Go button to load the URL that was entered in the input field
+               GoButton.onClick.AddListener(() => webView.Load(new Uri(URLField.text)));
 
-1. On the menu bar, select **Project > Properties > Configuration Properties > Debugging**.
+               // Subscribe to the Navigated event to update the URL input field whenever a navigation occurs
+               webView.Navigated += OnNavigated;
 
-1. Click the **Debugger to launch** drop-down and then select **Remote Machine**.
+               // Set the initial value of the URL input field to the current URL of the WebView
+               if (webView.Page != null)
+               {
+                  URLField.text = webView.Page.AbsoluteUri;
+               }
+         });
+      }
 
-1. In the **Machine Name** field, enter the IP address of your device, as shown below.
-   * You can find the IP address on your HoloLens 2 under **Settings > Network & Internet > Advanced Options**.
-   * We recommend that you manually enter your IP address rather than depend on the "Auto Detected" feature.
+      // Update the URL input field with the new path after navigation
+      private void OnNavigated(string path)
+      {
+         URLField.text = path;
+      }
 
-1. Set the **Authentication Type** to **Universal (Unencrypted Protocol)**:
+      // Enable or disable the Back button based on whether there's any history to go back to
+      private void CanGoBack(bool value)
+      {
+         BackButton.enabled = value;
+      }
+   }
+   ```
 
-   ![The Debugging settings in Project Settings dialog with Machine Name set](./hololens2-images/visualstudio_remote_ip.png)
 
-1. Click the **OK** button.
+1. Connect the UI GameObjects to the WebViewBrowser code that we just wrote:
+   1. Select the WebViewBrowser.
```

Comment:
```suggestion
   1. Select the **WebViewBrowser**.
```
if context is more clicking UI, bold.
if context is more code editor, backticks.